tailwind.config.js

```diff
@@ -63,5 +63,8 @@ module.exports = {
       },
     },
   },
-  plugins: [require("@tailwindcss/typography")],
+  plugins: [
+    require("tailwind-scrollbar")({ nocompatible: true }),
```

Comment:
`({ nocompatible: true })` defeated the purpose of the plugin

Reply:
working on it 🚀

Reply:
- Regarding why I use {nocompatible: true}

![Screenshot from 2024-03-04 10-53-31](https://github.com/coronasafe/leaderboard/assets/108052802/9071b5f7-c2a7-4d3d-aa48-74b691628e82)

At present, this package lacks any impact on Firefox; however, it will seamlessly integrate with Firefox upon its next release. Additionally, Firefox offers limited options for customizing the scrollbar interface.